core/application/src/tests/epoch_change.rs

```diff
@@ -122,25 +122,25 @@ async fn test_epoch_change_with_some_non_committee_nodes() {
         .execute_transaction_from_node(UpdateMethod::ChangeEpoch { epoch })
         .await;
     match result.unwrap_err() {
-        TransactionClientError::Reverted((_, TransactionReceipt { response, .. })) => {
+        ExecuteTransactionError::Reverted((_, TransactionReceipt { response, .. }, _)) => {
             assert_eq!(
                 response,
                 TransactionResponse::Revert(ExecutionError::NotCommitteeMember)
             )
         },
-        _ => panic!("unexpected error type"),
+        e => panic!("unexpected error type: {:?}", e),
```

Comment:
Teeny tiny nit, can inline all these panics with error printed

clippy lints this for prints so I'd assume they will for panics at some point anyway 

```suggestion
        e => panic!("unexpected error type: {e:?}"),
```